usng4j-api/src/main/java/org/codice/usng4j/CoordinatePrecision.java

```diff
@@ -64,16 +64,16 @@ public int getIntValue() {
     return this.precisionValue;
   }
 
-  public static CoordinatePrecision forEastNorth(int easting, int northing) {
-    int maxValue = Math.max(easting, northing);
+  public static CoordinatePrecision forEastNorth(String easting, String northing) {
+    int maxLength = Math.max(easting.trim().length(), northing.trim().length());
 
-    if (maxValue > 9999) {
+    if (maxLength > 4) {
```

Comment:
Would this be something like:
if(easting.trim().length() == 5 and northing.trim().length() == 5) {
     return CoordinatePrecision.ONE_METER;
}
Something similar for the rest of these precision checks.

Reply:
I'm a bit unsure of how to handle when easting/northing values of unequal lengths get passed in to this method. There are tests where it looks like invalid coordinates are being used (e.g. https://github.com/codice/usng4j/blob/7ea805a0f5e26b279c46fdc2383d5f095d8b41de/usng4j-impl/src/test/java/org/codice/usng4j/impl/CoordinateSystemTranslatorTest.java#L287-L288) that use the `CoordinatePrecision.forEastNorth()` method and are expected to pass. I think it makes sense to throw an exception when easting/northing values of unequal lengths are used, which I did inside the coordinate parsing methods, but there's not currently any checks inside the `CoordinatePrecision.forEastNorth()` method because that would cause those existing tests to fail. I'm not sure if there's something else I'm overlooking, but if there isn't, then we could change those tests to use valid easting/northing values and add the checks.